src/test/java/com/example/bookshop/repository/BookRepositoryTest.java

```diff
@@ -0,0 +1,79 @@
+package com.example.bookshop.repository;
+
+import com.example.bookshop.model.Book;
+import com.example.bookshop.model.Category;
+import com.example.bookshop.repository.book.BookRepository;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Set;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.test.context.jdbc.Sql;
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+public class BookRepositoryTest {
+    @Autowired
+    private BookRepository bookRepository;
+
+    @Sql(scripts = "classpath:database/category/book/insert-books.sql",
+            executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+    @Sql(scripts = "classpath:database/category/book/delete-from-books.sql",
+            executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)

```

Comment:
Can you move these scripts below class and avoid duplication?

Reply:
I need to run these scripts in certain order so if I do that, the method will break